Package.swift

```diff
@@ -41,7 +45,8 @@ let package = Package(
             dependencies: [
                 "VGSCollectSDK",
                 "LaunchDarkly",
-                "BasisTheoryElements"
+                "BasisTheoryElements",
+                "sentry"
```

Comment:
should/can this be capitalized?

Reply:
Was a typo. Fixed here 9b4a1157a9b82310fa770b83552353c5415b905d